pandas/core/generic.py

```diff
@@ -1126,7 +1124,9 @@ def to_hdf(self, path_or_buf, key, **kwargs):
             If applying compression use the fletcher32 checksum
         dropna : boolean, default False.
             If true, ALL nan rows will not be written to store.
-
+        data_columns : list of columns to create as data columns, or True to
+            use all columns.  If True and format='table', data will be written
```

Comment:
a ref to: http://pandas.pydata.org/pandas-docs/stable/io.html#query-via-data-columns

and ideally adding references as needed (e.g. compression) and for the format option